core/webcomponents/services/ELMSMediaService.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+/**
+ * Require the submission service.
+ */
+define('__ROOT__', dirname(dirname(dirname(__FILE__))));
+require_once('ELMSRestService.php');
+
+class ELMSMediaService extends ELMSRestService {
+
+  /**
+   * Get a list of projects
+   */
+  public function getMedia(array $options = NULL) {
+    $return = array();
+    $query = new EntityFieldQuery();
+    $query->entityCondition('entity_type', 'node');
+    /**
+     * @todo this should be configurable
+     */
+    $query->entityCondition('bundle', array('elmsmedia_image', 'external_video', 'svg', 'h5p_content', 'document', 'video', 'audio'));
```

Comment:
could probably do a not clause and then just ensure that cis_course and section don't get returned since they are data model based; everything else should be up for grabs. This most likely can be abstracted to your other class so that you can query from any of the systems via our REST engine any (generally) get the right response so we can start visualizing everything :) (gizmo type return would need some clean up / expansion obviously unless we added something into the sites themselves to say what types they are)